plugin/emacs/flycheck-quicklintjs.el

```diff
@@ -62,25 +53,22 @@
                :checker checker
                :end-pos (cdr region)))) (car (read-from-string output))))
 
-(flycheck-define-checker javascript-quicklintjs
+(flycheck-define-command-checker 'javascript-quicklintjs
   "quick-lint-js finds bugs in JavaScript programs.
 
 https://quick-lint-js.com"
-  :command ("quick-lint-js"
-            "--output-format=emacs-lisp"
-            (eval (let ((file (buffer-file-name)))
-                    (if file
-                      `("--path-for-config-search" ,file)
-                      ())))
```

Comment:
Did you intentionally rewrite this code in this commit? This change seems unrelated to unifying settings. (The change looks fine, though.)